sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/dev/AstPrinter.kt

```diff
@@ -107,3 +105,18 @@ fun textPointerAtOffset(psiDocument: Document, startOffset: Int): TextPointer {
 
     return DefaultTextPointer(startLineNumber + 1, startLineOffset)
 }
+
+private class DefaultTextPointer(val line: Int, val lineOffset: Int) : TextPointer {
+    override fun line() = line
+    override fun lineOffset() = lineOffset
+
+    override fun compareTo(other: TextPointer) =
```

Comment:
How about adding a few unit tests to increase coverage. On the one hand, it's not mandatory, on the other hand, this will be nice.

Btw, if this method is never used in our codebase, how about throwing `UnsupportedOperationException` instead of this implementation?

Reply:
You are right and actually I found we can simplify this even more by removing the dependency on the commons `TextRange` & `TextPointer`. There is no real need for it and it removes the methods we wouldn't be using anyway.